Documentation/botr/readytorun-format.md

```diff
@@ -225,22 +236,67 @@ additional data determined by the flags.
 | ``READYTORUN_FIELD_SIG_MemberRefToken``      |  0x10 | If set, the token is memberref token. If not set, the token is fielddef token. |
 | ``READYTORUN_FIELD_SIG_OwnerType``           |  0x40 | Field type. Typespec appended as additional data. |
 
-### READYTORUN_ IMPORT_SECTIONS::AuxiliaryData
+### READYTORUN_IMPORT_SECTIONS::AuxiliaryData
 
 For slots resolved lazily via ``READYTORUN_HELPER_DelayLoad_MethodCall`` helper, auxiliary data are 
-compressed argument maps that allow precise GC stack scanning while the helper is running.
+compressed argument maps that allow precise GC stack scanning while the helper is running. The CoreCLR runtime class [``GCRefMapDecoder``](https://github.com/dotnet/coreclr/blob/6b9a3d3a87825b1a34bd8f114c9b181ce75b3b2e/src/inc/gcrefmap.h#L157) is used to parse this information. This data would not be required for runtimes that allow conservative stack scanning.
+
+The auxiliary data table is supposed to contain the exact same number of GC ref map records as there are method entries in the import section. To accelerate GC ref map lookup, the auxiliary data section starts with a lookup table holding the offset of every 1024-th method in the runtime function table within the linearized GC ref map.
+
+| Offset in auxiliary data | Size | Content |
+|-------------------------:|-----:|:--------|
+| 0 | 4 | Offset to GC ref map info for method #0 relative to this byte i.e. 4 * (MethodCount / 1024 + 1) |
+| 4 | 4 | Offset to GC ref map info for method #1024 |
+| 8 | 4 | Offset to GC ref map info for method #2048 |
+| ... | | |
+4 * (MethodCount / 1024 + 1) | ... | Serialized GC ref map info |
+
+The GCRef map is used to encode GC type of arguments for callsites. Logically, it is a sequence ``<pos, token>`` where ``pos`` is 
+position of the reference in the stack frame and ``token`` is type of GC reference (one of [``GCREFMAP_XXX``](https://github.com/dotnet/coreclr/blob/6b9a3d3a87825b1a34bd8f114c9b181ce75b3b2e/src/inc/corcompile.h#L633) values):
+
+| CORCOMPILE_GCREFMAP_TOKENS | Value | Stack frame entry interpretation |
+|:---------------------------|------:|:--------|
+| ``GCREFMAP_SKIP`` | 0 | Not a GC-relevant entry |
```

Comment:
Is there a good reason to use double ticks for escaping? I would think a single tick should be enough. (This was in the original doc - but I have not noticed it.)

Reply:
I was under the impression that the double tick is supposed to be something like a "code quote" but perhaps I was mistaken. I started to learn md yesterday.

Reply:
In fact, in the GitHub PR rendering the "code quotes" look pretty annoying. I'm going to remove them everywhere.

Reply:
Fixed in 2nd commit (in tables anyway; in normal text they look good to me).

Reply:
The common markdown convention is to use single ticks to quote identifiers in the text and triple ticks to quote code fragments. I have looked at the other docs inside botr directory and all of them use single and triple ticks. I am not able to find any other instance of double ticks.

I think if you replace "``" with "`", everything is still going to render the same.

Reply:
Globally replaced in 10th commit.